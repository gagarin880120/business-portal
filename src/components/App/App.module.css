@font-face {
  font-family: Nunito-Regular;
  src: url('../../assets/fonts/Nunito-Regular.ttf');
}

.wrapper {
  font-family: Nunito-Regular;
}

.header {
  color: red;
}
