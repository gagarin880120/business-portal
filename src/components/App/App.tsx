import * as React from 'react';
import { hot } from 'react-hot-loader/root';
import styles from './App.module.css';

interface AppProps {
  name: string;
}

function App({ name }: AppProps) {
  return (
    <div className={styles.wrapper}>
      <h1 className={styles.header}>
        {`Hello, ${name}!`}
      </h1>
    </div>
  );
}

export default hot(App);
