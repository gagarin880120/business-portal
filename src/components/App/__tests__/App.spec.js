import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import App from '../App';
import { isExportDeclaration } from 'typescript';

describe('App', () => {
  describe('component', () => {
    test('should render without crashing', () => {
      const renderer = new ShallowRenderer();
      renderer.render(<App />);

      const result = renderer.getRenderOutput();

      expect(result).toMatchSnapshot();
    });
  });
});
