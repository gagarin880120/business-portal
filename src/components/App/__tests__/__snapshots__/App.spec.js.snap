// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should render without crashing 1`] = `
<div
  className="wrapper"
>
  <h1
    className="header"
  >
    Hello, undefined!
  </h1>
</div>
`;
